addon/components/amount-input/component.js

```diff
@@ -72,6 +75,17 @@ export default Component.extend({
   */
   update() {},
 
+  keyDown(event) {
+    if (event.keyCode === KEY_CODE_E) {
+      return false
+    } else if (
+      this.numberOfDecimal === 0 &&
+      event.keyCode === KEY_CODE_FULLSTOP
```

Comment:
Should we have the same behavior for comma (188)?

Reply:
',' is disallowed on input[type=number], at least on my machine :sweat_smile:

Reply:
Indeed I think it's because of the language of your browser. If - by coincidence - you switch to french, you could 😉

Reply:
:+1: will do

Reply:
thanks!